lib/pacemaker/pcmk_sched_group.c

```diff
@@ -616,8 +616,10 @@ pcmk__group_colocated_resources(const pe_resource_t *rsc,
     if (pe__group_flag_is_set(rsc, pe__group_colocated)
         || pe_rsc_is_clone(rsc->parent)) {
         /* This group has colocated members and/or is cloned -- either way,
-         * add every child's colocated resources to the list.
+         * add every child's colocated resources to the list. The first and last
+         * members will include the group's own colocations.
          */
+        colocated_rscs = g_list_append(colocated_rscs, (gpointer) rsc);
```

Comment:
Out of scope but I wonder if the order matters. Usually prepend is preferred if order doesn't matter. We use append in the shared (primitive) method too.

Reply:
It shouldn't, and the code doesn't appear to care, and no regression tests change, so I'll throw that in